bgpd/bgp_rpki.c

```diff
@@ -1350,15 +1356,17 @@ DEFPY (show_rpki_as_number,
 {
 	struct json_object *json = NULL;
 
+	if (uj)
+		json = json_object_new_object();
+
 	if (!is_synchronized()) {
-		if (!uj)
+		if (uj)
+			vty_json(vty, json);
+		else
 			vty_out(vty, "No Connection to RPKI cache server.\n");
 		return CMD_WARNING;
```

Comment:
is it possible to encapsulate error message in a json attribute ?

Reply:
Good point @pguibert6WIND, we might do this for all warning/error handling. Would be useful.

Reply:
I carry on in https://github.com/FRRouting/frr/pull/15034
https://github.com/FRRouting/frr/pull/15034/commits/1f1b83948da1890522f736c8ce7ad718793c0989 has been added for that